code/game/objects/items/food/misc.dm

```diff
@@ -650,6 +650,25 @@
 	tastes = list("peaches" = 1, "tin" = 7)
 	venue_value = FOOD_EXOTIC
 
+/obj/item/food/canned/tomatoes
+	name = "canned San Marzano tomatoes"
+	desc = "A can of premium San Marzano tomatoes, from the hills of Southern Italy."
```

Comment:
Should be something other than a real location, like Space Italy.

Reply:
Not everything needs to be from space, especially not something with a DOP.